src/main/webapp/app/course/tutorial-groups/services/tutorial-groups.service.ts

```diff
@@ -170,4 +170,27 @@ export class TutorialGroupsService {
             return tutorialGroup;
         }
     }
+
+    /**
+     * Export tutorial groups for a specific course to a CSV file.
+     *
+     * @param courseId the id of the course for which the tutorial groups should be exported
+     * @param fields   the list of fields to include in the CSV export
+     * @return an Observable containing the CSV file as a Blob
+     */
+    exportTutorialGroupsToCSV(courseId: number, fields: string[]): Observable<Blob> {
+        const params = { fields };
+        return this.httpClient.get(`${this.resourceURL}/courses/${courseId}/tutorial-groups/export/csv`, {
+            params,
+            responseType: 'blob',
+        });
+    }
+
+    exportToJson(courseId: number, fields: string[]): Observable<any> {
```

Comment:
Why is the return type `Observable<any>`, when we define a clear `responseType` in the `this.httpClient.get` call?

Reply:
adjusted it 👍